src/bp-core/classes/class-bp-invitation.php

```diff
@@ -132,6 +132,24 @@ class BP_Invitation {
 	 */
 	public $accepted;
 
+	/**
+	 * Columns in the invitations table.
+	 */
+	public static $columns = array(
```

Comment:
Concern for whoever doing code review

Doesn’t this vulnerability imply to all Database tables in WP?

Also can we not fetch the database table column instead of hardcoding column name here?
Ref:
https://dev-notes.eu/2016/02/wordpress-get-table-fieldnames/
IF we add more column in future then things might not work here.

Reply:
@sourabhmatolia I think its fine to have a static variable there.
If we can get this dynamically then one additional SQL query can perform to get the columns.
Instead of that when we modify this table it's fine to update the variable here.

```
global $wpdb;
$order_by_clean = array();
$table          = BP_Invitation_Manager::get_table_name();
$columns        = $wpdb->get_col( "DESC {$table}", 0 );
```
Let me know your thoughts here?